products/paas/build-deploy.md

```diff
@@ -57,3 +91,25 @@ shopware variable:create --level project --name env:COMPOSER_AUTH --json true --
 ```
 
 Make sure to replace `%place your key here%` with your actual token. You can find your token by clicking 'Install with Composer' in your Shopware Account.
+
+## Extending Shopware: Plugins & Apps
+
+The PaaS recipe uses the [Composer plugin loader](https://developer.shopware.com/docs/guides/hosting/installation-updates/cluster-setup#composer-plugin-loader).
+
+## Manually trigger rebuilds
+
+Sometimes you might want to trigger a rebuild and deploy of your environment without pushing new code to your project.
+
+Assuming you want to do this for your main environment, first create a REBUILD_DATE environment variable. This triggers a build right away to propagate the variable.
```

Comment:
Rephrase for clarity

_If you want to do this for your main environment, create a REBUILD_DATE environment variable first. This triggers a build right away to propagate the variable._